### DISCUSSION TITLE
issue: Single Check is not working in useFieldArray with Table

### DISCUSSION THREAD

**Discussion by rajagopal1997** [react-hook-form/react-hook-form#8561] [Q&A]
### Version Number

7.32.2

### Codesandbox/Expo snack

https://codesandbox.io/s/reacttableandreacthookformscheckbox-v1n301?file=/src/Columns.js

### Steps to reproduce

1. Please Check my [Sandboxcode](https://codesandbox.io/s/reacttableandreacthookformscheckbox-v1n301?file=/src/Columns.js)
2. Enter UOM code and UOM name fields then automatically row i'll be append
3. **isBaseUom** is Multiple checking but i want **single check**

### Expected behaviour

just i want single check in isBaseUom

https://user-images.githubusercontent.com/83446783/175474461-33d1de1d-238e-4a39-ac47-44a280a33c91.mp4

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Upinwv** [comment]
```
```

```
```

**Reply by NarenderSiri** [reply]
Not visible Answer

**Comment by rajagopal1997** [comment]
Unable to view your answer

**Comment by rajagopal1997** [comment]
Please this solution is too important for me.. i'm trying this to last 2 days

**Comment by bluebill1049** [comment]
I am more than happy to assist if the question is a bit clearer, and this is definitely not an issue that you have originally posted. 

> just i want single check in isBaseUom

What does that requirement mean? it's not clear to me and the shared codesandbox is messy with a lot of unrelated code. please update it and we can take a look at it again.
